content/en/releases/patch-releases.md

```diff
@@ -0,0 +1,188 @@
+
+End of Life for **1.19** is **2021-10-28**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
+|--- |--- |--- |
+| 1.19.11       | 2021-05-07           | 2021-05-12  |
+| 1.19.10       | 2021-04-09           | 2021-04-14  |
+| 1.19.9        | 2021-03-12           | 2021-03-17  |
+| 1.19.8        | 2021-02-12           | 2021-02-17  |
+| 1.19.7        | 2021-01-08           | 2021-01-13  |
+| 1.19.6        | [Tagging Issue](https://groups.google.com/g/kubernetes-dev/c/dNH2yknlCBA) | 2020-12-18  |
+| 1.19.5        | 2020-12-04           | 2020-12-09  |
+| 1.19.4        | 2020-11-06           | 2020-11-11  |
+| 1.19.3        | 2020-10-09           | 2020-10-14  |
+| 1.19.2        | 2020-09-11           | 2020-09-16  |
+| 1.19.1        | 2020-09-04           | 2020-09-09  |
+
+### 1.18
+
+**1.18** enters maintenance mode on **2021-04-28**
+
+End of Life for **1.18** is **2021-05-12**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
+|--- |--- |--- |
+| 1.18.19       | 2021-05-07           | 2021-05-12  |
+| 1.18.18       | 2021-04-09           | 2021-04-14  |
+| 1.18.17       | 2021-03-12           | 2021-03-17  |
+| 1.18.16       | 2021-02-12           | 2021-02-17  |
+| 1.18.15       | 2021-01-08           | 2021-01-13  |
+| 1.18.14       | [Tagging Issue](https://groups.google.com/g/kubernetes-dev/c/dNH2yknlCBA) | 2020-12-18  |
+| 1.18.13       | 2020-12-04           | 2020-12-09  |
+| 1.18.12       | N/A                  | 2020-11-12  |
+| 1.18.11       | [No-op release](https://groups.google.com/g/kubernetes-dev/c/nJix1xLQvZE) | 2020-11-11  |
+| 1.18.10       | 2020-10-09           | 2020-10-14  |
+| 1.18.9        | 2020-09-11           | 2020-09-16  |
+| 1.18.8        | N/A                  | 2020-08-13  |
+| 1.18.7        | 2020-08-07           | 2020-08-12  |
+| 1.18.6        | 2020-07-10           | 2020-07-15  |
+| 1.18.5        | 2020-06-25           | 2020-06-26  |
+| 1.18.4        | 2020-06-12           | 2020-06-17  |
+| 1.18.3        | 2020-05-15           | 2020-05-20  |
+| 1.18.2        | 2020-04-13           | 2020-04-16  |
+| 1.18.1        | 2020-04-06           | 2020-04-08  |
+
+## Non-Active Branch History
+
+These releases are no longer supported.
+
+| Minor Version | Final Patch Release | EOL date |
+| --- | --- | --- |
+| 1.17 | 1.17.17 | 2021-01-13 |
+| 1.16 | 1.16.15 | 2020-09-02 |
+| 1.15 | 1.15.12 | 2020-05-06 |
+| 1.14 | 1.14.10 | 2019-12-11 |
+| 1.13 | 1.13.12 | 2019-10-15 |
+| 1.12 | 1.12.10 | 2019-07-08 |
+| 1.11 | 1.11.10 | 2019-05-01 |
+| 1.10 | 1.10.13 | 2019-02-13 |
+| 1.9  | 1.9.11  | 2018-09-29 |
+| 1.8  | 1.8.15  | 2018-07-12 |
+| 1.7  | 1.7.16  | 2018-04-04 |
+| 1.6  | 1.6.13  | 2017-11-23 |
+| 1.5  | 1.5.8   | 2017-10-01 |
+| 1.4  | 1.4.12  | 2017-04-21 |
+| 1.3  | 1.3.10  | 2016-11-01 |
+| 1.2  | 1.2.7   | 2016-10-23 |
+
+[cherry-pick process]: https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md
+[release-managers]: /release-managers.md
```

Comment:
This will need to be replaced in the update-release-info script

Reply:
Updated with generated warning at top of page for now. I would also be willing to remove the pages and starting with less content. Thanks for the heads up!